### ISSUE TITLE
Federation font awesome works only with icons with class fas

### ISSUE THREAD

**Issue by klinux** [goauthentik/authentik#11104]
**Describe your question/**
We are trying to configure font awesome like google logo, but it uses class **fab**, like `<i class="fab fa-google"></i>`
And the code of authentik only configure fas class. There another way to configure font awesome?

**Relevant info**

**Screenshots**

**Logs**

**Version and Deployment (please complete the following information):**

-   authentik version: 2024.6
-   Deployment: Kubernetes

**Additional context**

**Comment by dannysauer** [comment]
https://github.com/goauthentik/authentik/blob/b3e89ef570bcda12e83dd148df1eafddf7ea1ecf/web/src/elements/AppIcon.ts#L82-L83

Commenting / upvoting because I would very much like to use actual logos without finding a random logo somewhere on the Internet as well. 😄

**Comment by thmo** [comment]
#19517 is related.

**Comment by dewi-tik** [comment]
I know this is an older issue but I'm closing in favor of the new issue because it has extra information

**Comment by dannysauer** [comment]
> [#19517](https://github.com/goauthentik/authentik/issues/19517) is related.

I thought I recognized that @thmo  username. I'm glad to see you're alive. I've given up on those docker pulp PRs, BTW. 😂

**Comment by thmo** [comment]
> I thought I recognized that [@thmo](https://github.com/thmo) username. I'm glad to see you're alive. I've given up on those docker pulp PRs, BTW. 😂

Yeah, sorry, I should clean up those repos up eventually 😉 - we nowadays mostly use the upstream images....

**Comment by dannysauer** [comment]
That was two jobs ago for me. I think "my" fork still get updated by automation, but I'm also pretty sure the company doesn't actually use it anymore. The repo was left up because it appeared in the upstream documentation for a while, IIRC.
